packages/cactus-core-api/src/main/json/openapi.json

```diff
@@ -67,15 +67,28 @@
                 "enum": [
                     "org.hyperledger.cactus.consensusalgorithm.PROOF_OF_AUTHORITY",
                     "org.hyperledger.cactus.consensusalgorithm.PROOF_OF_STAKE",
-                    "org.hyperledger.cactus.consensusalgorithm.PROOF_OF_WORK"
+                    "org.hyperledger.cactus.consensusalgorithm.PROOF_OF_WORK",
+                    "org.hyperledger.cactus.consensusalgorithm.SUBSTRATE"  
+                ],
+                "x-enum-varnames": [
+                    "Authority",
+                    "Stake",
+                    "Work",
+                    "Substrate"
```

Comment:
@RafaelAPB By the consesus algorithm family "Substrate" are you referring to all the consensus algorithms listed here on their documentation? https://docs.substrate.io/v3/advanced/consensus/#consensus-in-substrate

Reply:
Updated to Substrate-GRANPA (for clarity)